scan/src/utils/HexUtils.re

```diff
@@ -1,10 +1,9 @@
-let normalizeHexString = hexstr => {
-  let len = hexstr->String.length;
-  let prefix = len >= 2 ? hexstr->String.sub(0, 2) : "";
-
-  switch (prefix) {
-  | "0x"
-  | "0X" => hexstr->String.lowercase_ascii->String.sub(2, len - 2)
-  | _ => hexstr->String.lowercase_ascii
-  };
-};
+let normalizeHexString = hexstr =>
```

Comment:
Why do we need to change this?

Reply:
Just make it work for prefix 0x,0X and now we have \\x

Reply:
But it's not getting used any where. Also please add a good comment explaining what this function does. It's nontrivial for someone reading this now to understand the intention of this function.

Reply:
Also if you are changing this, then we can also change `GraphQLParser.hash` to not do the `substr`, but I feel like we should just not do this change.